.secrets.baseline

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
Less of a concern and more of just a gripe with the tool, I wish this could be YAML instead of JSON.